src/main/java/org/candlepin/subscriptions/tally/facts/FactNormalizer.java

```diff
@@ -217,6 +218,15 @@ private void normalizeSystemProfileFacts(
     getProductsFromProductIds(normalizedFacts, hostFacts.getSystemProfileProductIds());
   }
 
+  private void normalizeMarketplace(NormalizedFacts normalizedFacts, InventoryHostFacts hostFacts) {
+    if (!hostFacts.isMarketplace()) return;
+
+    if (hostFacts.getCores() != 0 || normalizedFacts.getCores() != 0) normalizedFacts.setCores(0);
+
+    if (hostFacts.getSockets() != 0 || normalizedFacts.getSockets() != 0)
```

Comment:
Would you mind putting braces around this guy?  I like having braces around conditionals that are multi-line to avoid errors like

```
if (something) 
  doAThing;
  expectedToBeConditionalButReallyIsnt;
```